src/Extensions.WebHosting/WebHostBuilderExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Bit.Extensions.Hosting;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace Bit.Extensions.WebHosting;
+
+public static class WebHostBuilderExtensions
+{
+    public static IWebHostBuilder UseBitwardenWebDefaults(this IWebHostBuilder webHostBuilder, Action<BitwardenWebHostOptions>? configure = null)
+    {
+        var bitwardenWebHostOptions = new BitwardenWebHostOptions();
+        configure?.Invoke(bitwardenWebHostOptions);
+        return webHostBuilder.UseBitwardenWebDefaults(bitwardenWebHostOptions);
+    }
+
+    public static IWebHostBuilder UseBitwardenWebDefaults(this IWebHostBuilder webHostBuilder, BitwardenWebHostOptions bitwardenWebHostOptions)
+    {
+        // TODO: Add services and default starting middleware
```

Comment:
ℹ️ Another topic for us to consider: [Serilog request logging](https://github.com/serilog/serilog-aspnetcore#request-logging).

Reply:
We can compare it against the built in one: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/http-logging/?view=aspnetcore-6.0